Add community link to readme (#9602)
netdata/netdata: CONTRIBUTING.md

We love to receive contributions. Maintaining a platform for monitoring everything imaginable requires a broad understanding of a plethora of technologies, systems and applications. We rely on community contributions and user feedback to continue providing the best monitoring solution out there.

There are many ways to contribute, with varying requirements of skills, explained in detail in the following sections.
Specific GitHub issues we need help with can be seen [here](https://github.com/netdata/netdata/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22help+wanted%22). Some of them are also labeled as "good first issue".

## All Netdata Users

### Give Netdata a GitHub star

This is the minimum open-source users should contribute back to the projects they use. Github stars help the project gain visibility, stand out. So, if you use Netdata, consider pressing that button. **It really matters**.

### Join the Netdata Community

> We have launched a discussion board where you can find many of us.

Reviewer: Link please

### Spread the word

Community growth allows the project to attract new talent willing to contribute. This talent is then developing new features and improves the project. These new features and improvements attract more users and so on. It is a loop. So, post about Netdata, present it to local meetups you attend, let your online social network or twitter, facebook, reddit, etc. know you are using it. **The more people involved, the faster the project evolves**.

### Provide feedback

> Is there anything that bothers you about Netdata? Did you experience an issue while installing it or using it? Would you like to see it evolve to you need? Let us know by making a post at [at the Netdata Community](https://community.netdata.cloud/category/4/feedback) to discuss it or [open a github issue](https://github.com/netdata/netdata/issues). Feedback is very important for open-source projects. We can't commit we will do everything, but your feedback influences [our road-map](https://community.netdata.cloud/category/1/announcements-and-roadmap) significantly. **We rely on your feedback to make Netdata better**.

Reviewer 2: Might be a typo here: Would you like to see it evolve to you need?

Also, should be "roadmap".

> Is there anything that bothers you about Netdata? Did you experience an issue while installing it or using it? Would you like to see it evolve to match your requirements? Let us know by making a post at [at the Netdata Community](https://community.netdata.cloud/category/4/feedback) to discuss it or [open a github issue](https://github.com/netdata/netdata/issues). Feedback is very important for open-source projects. We can't commit we will do everything, but your feedback influences [our roadmap](https://community.netdata.cloud/category/1/announcements-and-roadmap) significantly. **We rely on your feedback to make Netdata better**.

Reviewer 3: ```suggestion
Is there anything that bothers you about Netdata? Did you experience an issue while installing it or using it? Would you like to see it evolve to match your requirements? Let us know by making a post at [at Netdata Community](https://community.netdata.cloud/category/4/feedback) to discuss it or [open a GitHub issue](https://github.com/netdata/netdata/issues). Feedback is very important for open-source projects. We can't commit we will do everything, but your feedback influences [our roadmap](https://community.netdata.cloud/category/1/announcements-and-roadmap) significantly. **We rely on your feedback to make Netdata better**.
```

> #### Sponsor a backend

Reviewer 3: We should update this to mention and point to the exporting engine, not backends.

```suggestion
#### Sponsor an exporting connector
```

> We already support various [backends](backends) and we intend to support more. A "sponsor" for a backend:

Reviewer 2: Need a comma: backends, and we

> We already support various [backends](backends), and we intend to support more. A "sponsor" for a backend:

Reviewer 3: ```suggestion
We already support various [exporting connectors](/exporting/README.md), and we intend to support more. A "sponsor" for a connector:
```

> -   Suggests ways in which the information in Netdata could best be exposed to the particular backend, to facilitate meaningful presentation.

Reviewer 3: ```suggestion
-   Suggests ways in which the information in Netdata could best be exposed to the particular endpoint, to facilitate meaningful presentation.
```
